test/default-messages.js

```diff
@@ -1,37 +1,32 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('default messages', function (t) {
     t.plan(1);
 
-    var tc = tap.createConsumer();
+    var ps = spawn(process.execPath, [ __dirname + '/messages/defaults.js' ]);
```

Comment:
@ljharb, thanks for the tip on cross-platform compatibility. 👍 
do we need to remove _all_ forward slashes? 
or is it enough to go from:
```js
var ps = spawn(process.execPath, [ __dirname + '/messages/defaults.js' ]);
```
To:
```js
var ps = spawn(process.execPath, [ require('path').join( __dirname, 'messages/defaults.js') ]);
```

From @domenic's Gist: https://gist.github.com/domenic/2790533#paths-and-urls I _deduce_ that this should be enough.